src/components/patterns/elements/lockup/lockup.tsx

```diff
@@ -50,13 +50,13 @@ export const Lockup = async ({whiteText}: {whiteText?: boolean}) => {
 
   if (!lockupSettingsConfig?.suLockupEnabled) {
     return (
-      <Link href="/" className="flex flex-col text-black no-underline lg:flex-row lg:items-end">
-        <div className={twMerge("py-2 pr-9 lg:border-r-2", whiteText ? "border-white" : "border-black")}>
+      <Link href="/" className="flex flex-col text-black no-underline sm:flex-row sm:items-end">
+        <div className={twMerge("py-2 pr-9 sm:border-r-2", whiteText ? "border-white" : "border-black")}>
```

Comment:
oh there's a issue here. the result with either be `sm:border-r-2border-white` or `sm:border-r-2border-black` without any space between. Either use `clsx` or put in a space after the `-r-2`

Reply:
Ah! Great catch, Mike 😄 I actually need to clean this portion up since `whiteText` option is no longer needed with the Footer redesign (twas a bandaid for the last release since they wanted the lockup in sync for both the header and footer).